plugin/trino-hive/src/test/java/io/trino/plugin/hive/BaseTestHivePartitionsOnDataLake.java

```diff
@@ -55,12 +63,15 @@ protected QueryRunner createQueryRunner()
         this.dockerizedS3DataLake = closeAfterClass(
                 new HiveMinioDataLake(bucketName, ImmutableMap.of(), hiveHadoopImage));
         this.dockerizedS3DataLake.start();
+        this.hiveMetastore = this.dockerizedS3DataLake.getHiveHadoop().getHiveMetastoreClient();
         return S3HiveQueryRunner.create(
                 dockerizedS3DataLake,
                 ImmutableMap.<String, String>builder()
                         // This is required when using MinIO which requires path style access
                         .put("hive.insert-existing-partitions-behavior", "OVERWRITE")
                         .put("hive.non-managed-table-writes-enabled", "true")
+                        .put("hive.metastore-cache-ttl", "60m")
+                        .put("hive.metastore-refresh-interval", "10m")
```

Comment:
These are large, but not very large values.

- document why it's needed
- use values that are more closer to "infinity" (like 1d)